include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -1427,7 +1397,7 @@ __parallel_set_reduce_then_scan(oneapi::dpl::__internal::__device_backend_tag __
 
     return __parallel_transform_reduce_then_scan(
         __backend_tag, std::forward<_ExecutionPolicy>(__exec),
-        oneapi::dpl::__ranges::zip_view(
+        oneapi::dpl::__ranges::make_zip_view(
```

Comment:
Does this change really needed?

Reply:
I had this same question to Matt offline.  His response:

"In the first PR I went from make_zip_view -> zip_view: https://github.com/uxlfoundation/oneDPL/pull/1997#discussion_r1933086592, so the revert undid this."

It goes back to some issue with `make_zip_view` being able to process creation of a zip_view of zip_views I believe.  This returns a change that was necessary in the previous workaround PR. We still have that issue mentioned #1805, but this restores us to before the first workaround commit.

Reply:
OK